DEPS

```diff
@@ -118,7 +118,7 @@ allowed_hosts = [
 ]
 
 deps = {
-  'src': 'https://github.com/flutter/buildroot.git' + '@' + 'ad96f00615dc306c22cc09d433eee50392f8033c',
+  'src': 'https://github.com/zanderso/buildroot.git' + '@' + '37f0ef84bfd2121d126cd6226d440e8228a79a1a',
```

Comment:
Will this be updated after https://github.com/flutter/buildroot/pull/645 is landed?

Reply:
Done.